src/Search.js

```diff
@@ -1,23 +1,37 @@
-import React from "react";
-
-const Search = () => {
+import React, { useState } from "react";
+import SearchButton from "./SearchButton";
+const Search = ({ search }) => {
+  const [searchInput, setSearchInput] = useState("");
+  function handleInputChange(event) {
+    setSearchInput(event.target.value);
+    console.log(event.target.value);
+  }
+  function handleSubmit(event) {
+    event.preventDefault();
+    search(searchInput);
+    // setSearchInput(event.target.value);
+    console.log(searchInput);
+    setSearchInput("");
```

Comment:
Very good implementation of search input! You even thought about resetting the input value after submitting👏

More console logs and commented out code in this file - please delete it🙂